### PR TITLE
ENCD-5850-fix-analysis-title-and-permission

### PR DESCRIPTION
**Pull request by yunhailuo**

### CODE REVIEW

**Review by jnau — APPROVED** [review]

**Review by jnau — CHANGES_REQUESTED** [review]
code change looks fine. Indexing tests failing. Looks like something in tests needs to be changed as well

**Review by forresttanaka — DISMISSED** [review]
It looks good.

**Review by jnau — DISMISSED** [review]
tests passed and code changes look good

File: src/encoded/schemas/analysis.json [hunk 1]
```diff
@@ -13,6 +13,7 @@
         { "$ref": "mixins.json#/uuid" },
         { "$ref": "mixins.json#/accession" },
         { "$ref": "mixins.json#/aliases" },
+        {"$ref": "mixins.json#/attribution" },
```
Comment (keenangraham) [inline src/encoded/schemas/analysis.json]: Extra space in front for consistency.

File: src/encoded/tests/data/inserts/expected_metadata.tsv [hunk 2]
```diff
@@ -84,9 +84,9 @@ ENCFF333FSQ	fastq	fastq		reads		ENCSR751ISO	long read RNA-seq	NTR:0000637	techni
...
-ENCFF003MRN	bam	bam		alignments	GRCh38	ENCSR123MRN	microRNA-seq	UBERON:0001891	midbrain	tissue	Homo sapiens	dexamethasone	60 nM	8 hour								miRNA					unstranded	2015-08-31	Roadmap			<30	1	1_1							/files/ENCFF001MRN/	20	ENCODE Processing Pipeline	94be74b6e11515390000f4ebfa66d78d		http://localhost/files/ENCFF003MRN/@@download/ENCFF003MRN.bam	V24	Illumina Genome Analyzer II		released		Lab custom v10.20.30 GRCh38 V24	in progress		missing documents	incomplete analysis_steps, inconsistent library biosample, mixed biosample perturbations
+ENCFF003MRN	bam	bam		alignments	GRCh38	ENCSR123MRN	microRNA-seq	UBERON:0001891	midbrain	tissue	Homo sapiens	dexamethasone	60 nM	8 hour								miRNA					unstranded	2015-08-31	Roadmap			<30	1	1_1							/files/ENCFF001MRN/	20	ENCODE Processing Pipeline	94be74b6e11515390000f4ebfa66d78d		http://localhost/files/ENCFF003MRN/@@download/ENCFF003MRN.bam	V24	Illumina Genome Analyzer II		released		Mixed pipelines v10.20.30 GRCh38 V24	in progress		missing documents	incomplete analysis_steps, inconsistent library biosample, mixed biosample perturbations
```
Comment (keenangraham) [inline src/encoded/tests/data/inserts/expected_metadata.tsv:87]: Looks like expected annotation metadata also needs to be updated. Test failing.

**Review by keenangraham — CHANGES_REQUESTED** [review]

**Review by keenangraham — DISMISSED** [review]

**Comment by keenangraham** [comment]
Make sure to rebase this over dev before merging. Some of the expected metadata tests might have to be updated now that SNO-197 is merged.

**Review by keenangraham — APPROVED** [review]

**Pull request merged**